google-cloud-spanner/lib/google/cloud/spanner/project.rb

```diff
@@ -452,7 +461,11 @@ def create_database instance_id, database_id, statements: []
         #   * Label values must be between 0 and 63 characters long and must
         #     conform to the regular expression `([a-z]([-a-z0-9]*[a-z0-9])?)?`.
         #   * No more than 64 labels can be associated with a given resource.
-        #
+        # @param [Boolean] enable_resource_based_routing Enable/Disable
+        #   resource-based routing for data operation, by default it is
+        #   disabled. Resource based routing can be enabled using the
+        #   environment `GOOGLE_CLOUD_ENABLE_RESOURCE_BASED_ROUTING` to
```

Comment:
GOOGLE_CLOUD_ENABLE_RESOURCE_BASED_ROUTING needs to be GOOGLE_CLOUD_SPANNER_ENABLE_RESOURCE_BASED_ROUTING

Reply:
done